src/Vizkit3dWorld.cpp

```diff
@@ -270,11 +287,9 @@ void Vizkit3dWorld::setCameraPose(base::samples::RigidBodyState pose) {
      * - Y left
      * - Z up
      */
-    osg::Matrixd m = poseMatrix *
-                     osg::Matrixd::rotate(M_PI_2, osg::Vec3(0.0, 0.0, 1.0)) *
-                     osg::Matrixd::rotate(-M_PI_2, osg::Vec3(1.0, 0.0, 0.0));
-
-
+    osg::Matrixd m = poseMatrix* 
```

Comment:
Again here...nothing related to the model exclusion, but with the process of SDF updating.
Both visual exclusion and that transformation is related to handle with the changes done in the sdf model. Is it recommended to open a different PR to deal with it?